mods/creative/inventory.lua

```diff
@@ -25,7 +25,9 @@ function creative.init_creative_inventory(player)
 	player_inventory[player_name] = {
 		size = 0,
 		filter = "",
-		start_i = 0
+		start_i = 0,
+		_old_filter = nil,
+		_old_content = nil
```

Comment:
I get the idea but why not just `old_filter` and `old_content`

Reply:
To denote they're for caching only, and not thought to be changed anywhere else in the code.

Reply:
Sure but I don't see other code using `_` variable names. IMO putting a comment there suffices.